app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, distance: int) -> Distance:
+        distance3 = Distance(distance + self.km)
+        return distance3
+
+    def __iadd__(self, distance: int) -> Distance:
+        self.km = Distance(distance + self.km)
+        return self
+
+    def __mul__(self, number: float) -> Distance:
+        self.km = self.km * number
+        return self
+
+    def __truediv__(self, number: float) -> Distance:
+        self.km = round(self.km / number, 2)
+        return self
+
+    def __lt__(self, distance: int) -> bool:
+        if self.km < distance:
+            return True
+        else:
+            return False
+
+    def __gt__(self, distance: int) -> bool:
+        if self.km > distance:
+            return True
+        else:
+            return False
```

Comment:
the same, distance could be the Distance class ex., so u should check for this case, also unnecessary else here